src/nvim/getchar.c

```diff
@@ -2025,8 +2023,6 @@ static int8_t get_key_from_user(int *cp, int *timedoutp, int *mode_deletedp,
    * place does not matter.
    */
   int bytes_read = 0;
-  int new_wcol = curwin->w_wcol;
-  int new_wrow = curwin->w_wrow;
   if (       advance
       && typebuf.tb_len == 1
       && typebuf.tb_buf[typebuf.tb_off] == ESC
```

Comment:
(A bit below, not your changes.)

A function that both modifies `curwin->w_w(col|row)` and is supposed to return a new version of it? I'm surprised you actually cut through the full logic of this :).